src/EditorFeatures/CSharpTest/ConvertNamespace/ConvertNamespaceCommandHandlerTests.cs

```diff
@@ -70,8 +70,7 @@ class C
 
 class C
 {
-}
-");
+}");
```

Comment:
All of these tests were actually verifying the incorrect behaviour, so I've updated them, but added new tests for this issue specifically just in case.